telemetry/telemetrybuffer.go

```diff
@@ -303,8 +303,15 @@ func ReadConfigFile(filePath string) (TelemetryConfig, error) {
 
 // ConnectToTelemetryService - Attempt to spawn telemetry process if it's not already running.
 func (tb *TelemetryBuffer) ConnectToTelemetryService(telemetryNumRetries, telemetryWaitTimeInMilliseconds int) {
-	path, dir := getTelemetryServiceDirectory()
+
```

Comment:
Let's remove the extra lines(whitespaces) I guess that's why linter is complaining.

Reply:
Will do.